Mathlib/Topology/GDelta/Basic.lean

```diff
@@ -149,7 +149,7 @@ theorem IsGδ.iUnion [Finite ι'] {f : ι' → Set X} (h : ∀ i, IsGδ (f i)) :
 
 end IsGδ
 
-section residual
+section Residual
```

Comment:
The corresponding definition is named `residual` (in lowercase) --- I'd leave the section as-is.